### PR TITLE
Kiali fixes

### PR DESCRIPTION
**Pull request by bartusz01**
Will fix issues:
- #730 
- kiali errors with jaeger tracing
- cspell complaining about valid word 'renamings'

### CODE REVIEW

**Review by j-zimnowoda — APPROVED** [review]
LGTM. Especially that Kiali does not need to use public URLs in order to access prometheus 👌

File: values/kiali-operator/kiali-operator.gotmpl [hunk 1]
```diff
@@ -71,19 +71,18 @@ cr:
         cache_duration: 10
         cache_enabled: true
         cache_expiration: 300
-        in_cluster_url: http://po-prometheus.monitoring:9090
```
Comment (Morriz) [inline values/kiali-operator/kiali-operator.gotmpl:74]: why is this one removed? they should both exist to allow public domain access

**Review by Morriz — CHANGES_REQUESTED** [review]
tnx for the fix...one small thing

File: values/kiali-operator/kiali-operator.gotmpl (hunk 1 above)
Comment (bartusz01) [reply values/kiali-operator/kiali-operator.gotmpl:74]: It's not a valid flag, see [here](https://github.com/kiali/kiali-operator/blob/v1.39/deploy/kiali/kiali_cr.yaml) as reference

Comment (j-zimnowoda) [reply values/kiali-operator/kiali-operator.gotmpl:74]: @Morriz does it answer your question?

**Review by Morriz — APPROVED** [review]

**Pull request merged**